daprdocs/content/en/operations/security/mtls.md

```diff
@@ -246,6 +257,14 @@ Once Sentry has been completely restarted run:
 kubectl rollout restart deploy/dapr-operator -n <DAPR_NAMESPACE>
 kubectl rollout restart statefulsets/dapr-placement-server -n <DAPR_NAMESPACE>
 ```
+{{% alert title="Recommended CLI command to achieve above steps" color="success" %}}
```

Comment:
It is hard to separate out the kubectl approach and the Dapr CLI approach here. Can you add headings for each so that these are separated? For example "Custom root certificate upgrade using kubectl" and "Custom root certificate upgrade using CLI"

Reply:
Also my concern here, is that given the CLI is the preferred method we should have this first BEFORE the kubectl method. So could you switch this around in the order?